client/components/promo-section/promo-card/cta.tsx

```diff
@@ -25,6 +25,7 @@ export interface CtaButton {
 export interface Props {
 	cta: CtaButton;
 	learnMoreLink?: CtaAction | null;
+	featureIncludedInPlan?: boolean;
```

Comment:
It might be nice to include some docs here to explain what this does. Something like the following:

```suggestion
    /**
    * Includes "included in your current plan" if true or "not included in your current plan" if false.
    *
    * This option is ignored if `learnMoreLink` is set.
    */
	featureIncludedInPlan?: boolean;
```

Reply:
I had added some docs further down in the file where the logic is handled, but could also move it up here near the param/prop definition. 

https://github.com/Automattic/wp-calypso/pull/81915/files#diff-06c245e990d0c293b17599f9a606e1293b05c19a6080496dc22509d2a0d86c5bR88

Reply:
The nice thing about having a comment directly in the props list is that it will show up in Language Servers and IDEs when someone uses the prop or looks at the component.